src/tests/lockup.spec.ts

```diff
@@ -20,10 +21,32 @@ test("view all info about lockup account on 'finality' block", async (t) => {
 });
 
 test("view all info about lockup account on particular block", async (t) => {
+  const arhivalConfig = {
+    nodeUrl: "https://archival-rpc.mainnet.near.org",
+    networkId: "mainnet",
+    keyStore: connectOptions.keyStore,
+    headers: connectOptions.headers
+  };
+
   const res = await viewLockupAccount(
     lockupDataMock[1].lockupAccountId,
-    null,
+    arhivalConfig,
```

Comment:
Now this and the next test are almost identical. I suggest merge them and have a loop over the "lockupDataMock" array (see other comment about "lockupDataMock")